frontend/views/containers/sidebar/Profile.vue

```diff
@@ -26,27 +30,46 @@
 @import "../../../assets/sass/theme/index";
 
 .c-profile {
-  background: $body-background-color;
+  background-color: $primary-bg-a;
   padding: $gi-spacer-sm;
-  border-radius: $radius;
-  margin-top: $gi-spacer;
+  margin-top: 0.75rem;
 }
 
 .c-user {
   max-width: 5rem;
   white-space: nowrap;
-  line-height: 1;
+  line-height: 1.15rem;
+
+  span {
+    color: $text-light;
+  }
+}
+
+// https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors
+/deep/ .button {
```

Comment:
According to that link, `/deep/` is an alias for `>>>`, which is put in between something like so:

```
.a >>> .b { /* ... */ }
```

So is having it at the front of the selector correct usage?

Reply:
It's an alias yes. It tell PostCSS to add css properties to child element that are normally outside of the parent scope
For more explanation, please read the doc you asked me to add in comment: https://vue-loader.vuejs.org/guide/scoped-css.html

Reply:
@pieer I did read that link, and that's what prompted my question above. In that link they only ever use `>>>` in between selectors, whereas here it's not being placed between any selectors, hence my confusion (since this usage doesn't seem supported by the docs).

Reply:
Yes it's working. It tell PostCSS to add css properties to child element that are normally outside of the parent scope.